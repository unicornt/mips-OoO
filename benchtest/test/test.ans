00000018
0000000a
0000000a