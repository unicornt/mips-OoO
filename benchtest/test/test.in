addi $v0,$0,10
addi $v1,$0,11
sw $v0, 0($0)
sw $v0, 64($0)
sw $v0, 128($0)
sw $v0, 192($0)
sw $v1, 0($0)
nop
nop
nop
nop