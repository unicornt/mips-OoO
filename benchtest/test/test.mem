2002000a
2003000b
ac020000
ac020040
ac020080
ac0200c0
ac030000
00000000
00000000
00000000
