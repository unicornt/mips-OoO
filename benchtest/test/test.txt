RAM[0] <= 32'h2002000a;
RAM[1] <= 32'h2003000b;
RAM[2] <= 32'hac020000;
RAM[3] <= 32'hac020040;
RAM[4] <= 32'hac020080;
RAM[5] <= 32'hac0200c0;
RAM[6] <= 32'hac030000;
RAM[7] <= 32'h00000000;
RAM[8] <= 32'h00000000;
RAM[9] <= 32'h00000000;
