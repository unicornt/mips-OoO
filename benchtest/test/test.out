 0x0 : addi $v0,$0,10       | 2002000a
 0x4 : addi $v1,$0,11       | 2003000b
 0x8 : sw $v0, 0($0)        | ac020000
 0xc : sw $v0, 64($0)       | ac020040
0x10 : sw $v0, 128($0)      | ac020080
0x14 : sw $v0, 192($0)      | ac0200c0
0x18 : sw $v1, 0($0)        | ac030000
0x1c : nop                  | 00000000
0x20 : nop                  | 00000000
0x24 : nop                  | 00000000
0x28 : nop                  | 
